library/src/test/kotlin/wu/seal/jsontokotlin/JsonToKotlinBuilderTest.kt

```diff
@@ -1,12 +1,101 @@
+                { "firstName": "Tad", "lastName": "Williams", "genre": "fantasy" },
+                { "firstName": "Frank", "lastName": "Peretti", "genre": "christian fiction" }
+                ],
+                "musicians": [
+                { "firstName": "Eric", "lastName": "Clapton", "instrument": "guitar" },
+                { "firstName": "Sergei", "lastName": "Rachmaninoff", "instrument": "piano" }
+                ] } """
+
+        val actualOutput = JsonToKotlinBuilder()
+                .setPackageName("com.my.package.name")
+                .enableVarProperties(false) // optional, default : false
+                .setPropertyTypeStrategy(PropertyTypeStrategy.AutoDeterMineNullableOrNot) // optional, default :  PropertyTypeStrategy.NotNullable
+                .setDefaultValueStrategy(DefaultValueStrategy.AvoidNull) // optional, default : DefaultValueStrategy.AvoidNull
+                .setAnnotationLib(TargetJsonConverter.MoshiCodeGen) // optional, default: TargetJsonConverter.None
+                .enableComments(true) // optional, default : false
+                .enableOrderByAlphabetic(true) // optional : default : false
+                .enableInnerClassModel(true) // optional, default : false
+                .enabelMapType(true)// optional, default : false
+                .enableCreateAnnotationOnlyWhenNeeded(true) // optional, default : false
+                .setIndent(4)// optional, default : 4
+                .setParentClassTemplate("android.os.Parcelable") // optional, default : ""
+                .enableKeepAnnotationOnClass(true) // optional, default : false
+                .enableKeepAnnotationOnClassAndroidX(true) // optional, default : false
+                .enableAnnotationAndPropertyInSameLine(true) // optional, default : false
+                .enableParcelableSupport(true) // optional, default : false
+                .setPropertyPrefix("MyPrefix") // optional, default : ""
+                .setPropertySuffix("MySuffix") // optional, default : ""
+                .setClassSuffix("MyClassSuffix")// optional, default : ""
+                .enableForceInitDefaultValueWithOriginJsonValue(true) // optional, default : false
+                .enableForcePrimitiveTypeNonNullable(true) // optional, default : false
+                .build(json1, "GlossResponse") // finally, get KotlinClassCode string
+
+
+        val expectedOutput = """
+            package com.my.package.name
+
+            import com.squareup.moshi.Json
+            import com.squareup.moshi.JsonClass
+            import android.os.Parcelable
+            import android.support.annotation.Keep
+            import androidx.annotation.Keep
+            import kotlinx.android.parcel.Parcelize
+            import android.os.Parcelable
+
+            @SuppressLint("ParcelCreator")
+            @Parcelize
+            data class GlossResponseMyClassSuffix(
+                val MyPrefixAuthorsMySuffix: List<AuthorMyClassSuffix> = listOf(),
```

Comment:
There is no Moshi(CodeGen)  Annotation here, Maybe there are some bugs in this program

Reply:
I think this option may lead this bug: `enableCreateAnnotationOnlyWhenNeeded` how about let it stay last.

Reply:
Or we could open a new issue for this problem, then we fix it later.

Reply:
Okay

Reply:
I think this got fixed in #260 , Isn't it ?

Reply:
yes